application/gui/CMakeLists.txt

```diff
@@ -2,10 +2,11 @@ add_library(gui INTERFACE)
 target_sources(gui INTERFACE    ${CMAKE_CURRENT_SOURCE_DIR}/gui_utils.hpp 
                                 ${CMAKE_CURRENT_SOURCE_DIR}/grid_display.hpp 
                                 ${CMAKE_CURRENT_SOURCE_DIR}/shortcuts.hpp 
-                                ${CMAKE_CURRENT_SOURCE_DIR}/menu.hpp)
+                                ${CMAKE_CURRENT_SOURCE_DIR}/menu.hpp
+                                ${CMAKE_CURRENT_SOURCE_DIR}/detections_menu.hpp)
```

Comment:
panel_renderer.hpp seems to be missing, unless it's not a part of the interface, but some private stuff